app/javascript/hooks/mutations/admin/manage_users/useAdminCreateUser.jsx

```diff
@@ -7,18 +7,9 @@ export default function useAdminCreateUser({ onSettled }) {
   const { t } = useTranslation();
   const queryClient = useQueryClient();
 
-  const addInferredLanguage = (data) => {
```

Comment:
Users created by a users manager will have the users manager registered language by default unlike now they will default to the manager browser language (which may differ from their language on Greenlight).